source/quick-start.txt

```diff
@@ -13,112 +20,146 @@ Quick Start
+Once you configure your dependencies, ensure that they are available to your
+project by running the dependency manager and refreshing the
+project in your IDE.
+
+Add Serialization Library Dependencies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To enable the driver to convert between Kotlin objects and BSON, the
+data format for documents in MongoDB, you must also add one or both of the
+following serialization packages to your application:
+
+- ``bson-kotlinx`` *(Recommended)*
+- ``bson-kotlin``
+
+If you are using Gradle to manage your packages, add one of the following
+entries to your ``build.gradle.kts`` dependencies list: 
+
+.. include:: /includes/serialization-libs-gradle-versioned.rst
+
+If you are using Maven to manage your packages, add one of the following
+entries to your ``pom.xml`` dependencies list:
+
+.. include:: /includes/serialization-libs-maven-versioned.rst
+
+Once you configure your dependencies, ensure that they are available to your
+project by running the dependency manager and refreshing the
+project in your IDE.
+
+To learn more about these packages, see
+:ref:`fundamentals-kotlin-serialization`.
 
 Create a MongoDB Cluster
 ------------------------
 
 After setting up your Kotlin project dependencies, create a MongoDB cluster
-where you can store and manage your data. Complete the
-:atlas:`Get Started with Atlas </getting-started?jmp=docs_driver_kotlin>` guide
+in which you can store and manage your data. Complete the
+:atlas:`Get Started with Atlas </getting-started?jmp=docs_driver_kotlin>` tutorial
 to set up a new Atlas account, create and launch a free tier MongoDB cluster,
-load datasets, and interact with the data.
+and load sample datasets.
 
-After completing the steps in the Atlas guide, you should have a new MongoDB
-cluster deployed in Atlas, a new database user, and sample datasets loaded
-into your cluster.
+Once you complete the steps in the Get Started with Atlas tutorial, you
+have a new MongoDB cluster deployed in Atlas, a new database user, and
+sample data loaded into your cluster.
 
 Connect to your Cluster
 -----------------------
 
-In this step, we create and run an application that uses the MongoDB Kotlin
-driver to connect to your MongoDB cluster and run a query on the sample data.
+This step shows how to create and run an application that uses the
+{+driver-short+} to connect to your MongoDB cluster and run a query on
+the sample data.
 
-We pass instructions to the driver on how to connect to your
-MongoDB cluster in a string called the *connection string*. This string
-includes information on the hostname or IP address and port of your
-cluster, authentication mechanism, user credentials when applicable, and
+First, you must specify how the driver connects to your MongoDB cluster
+by including a *connection string* in your code. This string includes
+information on the hostname or IP address and port of your cluster,
+authentication mechanism, user credentials, if applicable, and
```

Comment:
Since this is essentially a list, the "if applicable" seems like its own list item and tripped me up when reading it (since it's separated by a comma). I think we would probably be okay removing "if applicable" altogether to make things more clear.

```suggestion
by including a *connection string* in your code. This string includes
information on the hostname or IP address and port of your cluster,
authentication mechanism, user credentials, and
```